src/bot.py

```diff
@@ -109,10 +113,10 @@ def generate_batch(self):
         batch = ''
         counter = 0
         while counter != limit:
-            batch += random.choice(config['emojis'])
+            batch += random.choice(emojis)
             counter += 1
-        batch = batch[:16] # Temporary hack
-        tweet = '\n'.join(textwrap.wrap(batch, config['limit_per_line']))
+            if counter > 16: break
+        tweet = '\n'.join(textwrap.wrap(batch, int(config['formatting']['limit_per_line'])))

```

Comment:
Could you document this change a bit better?

Reply:
I made the loop break after 16 times because **batch = batch[:16]** would keep only the first 16th items of that list anyway.

Reply:
What'd happen if I randomly decided to manipulate the following configurations?

```
lines = 4
limit_per_line = 4
```

Reply:
Yes, as batch = batch[:16] did. I just re wrote it in a much simple way.

Reply:
You're right! Thanks for staying true to the original implementation, despite that being a temporary hack :P